.travis.yml

```diff
@@ -0,0 +1,34 @@
+# The stanford_travisci_scripts repository contains external scripts
+# used here to configure server and build a self-service site.
+# https://github.com/SU-SWS/stanford_travisci_scripts
+
+language: php
+php:
+  - 5.6
+
+mysql:
+  database: drupal
+  username: root
+  encoding: utf8
+
+before_install:
+  # download external repository with Stanford Travis CI scripts
+  - git clone https://github.com/SU-SWS/stanford_travisci_scripts.git
+  - chmod +x stanford_travisci_scripts/bin/*.sh
+  # install composer packages
+  - stanford_travisci_scripts/bin/before_install.sh
+
+install:
+  # install self-service site, disable webauth module, start webserver and selenium webdriver
+  - export PATH="$HOME/.composer/vendor/bin:$PATH"
+  - stanford_travisci_scripts/bin/install.sh
+
+before_script:
+  # download linky_clicky, grab related tests, and start xvfb virtual display
+  - stanford_travisci_scripts/bin/before_script.sh
+
+script:
+  # make sure shell can find drush and run behat tests here, so Travis CI reports accurate results
+  - export PATH="$HOME/.composer/vendor/bin:$PATH"
+  - cd stanford_travisci_scripts
+  - bin/behat
```

Comment:
This should be a script as well. We may want to change how we run behat in the future.